stackit/internal/services/argus/instance/resource.go

```diff
@@ -1229,12 +1305,49 @@ func getMockAlertConfig(ctx context.Context) (alertConfigModel, error) {
 		return alertConfigModel{}, fmt.Errorf("mapping route: %w", core.DiagsToError(diags))
 	}
 
+	mockGlobalConfig, diags := types.ObjectValue(globalConfigurationTypes, map[string]attr.Value{
+		"opsgenie_api_key":   types.StringNull(),
+		"opsgenie_api_url":   types.StringNull(),
+		"resolve_timeout":    types.StringValue("5m"),
```

Comment:
Is this on purpuse?

Reply:
Yes, I am matching the default configuration on the API side. But I forgot the `smtp_from`, so that's fixed now. The others need to be explicitely null, otherwise it will error in the mapping methods